### PR TITLE
Update secondaryBrowser.ts

### PR DESCRIPTION
**Pull request by mobajemu-msft**

### CODE REVIEW

**Review by AE-MS — APPROVED** [review]

**Comment by AE-MS** [comment]
You'll have to create a changefile by running `pnpm changefile`

File: packages/teams-js/src/public/secondaryBrowser.ts [hunk 1]
```diff
@@ -6,14 +6,16 @@ import { ErrorCode } from './interfaces';
...
- * For e.g., opening a URL in the Host App inside a browser
+ * Namespace to power up the in-app browser experiences in the host app.
+ * For e.g., opening a URL in the host app inside a browser
  *
  * @beta
  */
 export namespace secondaryBrowser {
   /**
-   * Open a URL in the secondary browser aka in-app browser
+   * Open a URL in the secondary browser
```
Comment (erikadoyle) [inline packages/teams-js/src/public/secondaryBrowser.ts:16]: Line breaks in source don't necessarily render in the published doc, so these three lines will probably publish as a run-on sentence. Could you add punctuation?

**Review by erikadoyle — CHANGES_REQUESTED** [review]
Minor change I just noticed, otherwise looks good

File: change/@microsoft-teams-js-4885f2ff-afee-4756-9192-4731c184d1ba.json
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-4885f2ff-afee-4756-9192-4731c184d1ba.json:1]: You probably shouldn't be deleting an existing changefile

**Review by TrevorJoelHarris — CHANGES_REQUESTED** [review]
It looks like you're deleting someone else's change, which is probably not what you intended to do

File: change/@microsoft-teams-js-4885f2ff-afee-4756-9192-4731c184d1ba.json
Comment (mobajemu-msft) [reply change/@microsoft-teams-js-4885f2ff-afee-4756-9192-4731c184d1ba.json:1]: yea looks like something went wrong. thanks for the callout. will fix that

**Comment by mobajemu-msft** [comment]
Third time is the charm #1889

**Pull request closed without merging**